nucliadb_node/src/writer/grpc_driver.rs

```diff
@@ -24,8 +24,8 @@ use nucliadb_protos::{
     op_status, DeleteGraphNodes, EmptyQuery, EmptyResponse, OpStatus, Resource, ResourceId,
     SetGraph, ShardCleaned, ShardCreated, ShardId, ShardIds, VectorSetId, VectorSetList,
 };
+use nucliadb_telemetry::async_telemetry::send_telemetry_event;
```

Comment:
`async_telemetry` module name is a bit redundant IMO.
Would you mind removing the `telemetry` suffix ?

Reply:
Since async is a reserved keyword, I have ruled in favor of `sync` and `blocking`.